libvisual/tools/lv-tool/lv-tool.cpp

```diff
@@ -455,6 +459,39 @@ namespace {
       }
   };
 
+  class ResizeRequest
+  {
+      unsigned int m_width;
+      unsigned int m_height;
+      LV::Time m_received_at;
+      LV::Time m_applied_at;
+
+  public:
+      ResizeRequest() : m_width(0), m_height(0) {
+      }
+
+      unsigned int width() const { return m_width; }
+      unsigned int height() const { return m_height; }
+
+      bool is_due() const {
+          if (m_applied_at >= m_received_at) {
+              return false;
+          }
+          const double seconds_passed = (LV::Time::now() - m_received_at).to_usecs() / 1000.0 / 1000.0;
+          return seconds_passed >= 0.2; // >=300ms is known to feel slow to humans
```

Comment:
It would be better to backport `LV::Time::to_secs()` and use that instead.

Reply:
@kaixiong good idea, done